anaconda_updates/anaconda_updates/releases/__init__.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, branch_type,
                  pykickstart_args=[],
                  simpleline_args=[],
                  dasbus_args=[],
+                 meh_args=[],
```

Comment:
Maybe this could use a refactor too? `proj_args={"meh": "blah"}` etc.

Reply:
Fixed by a complete removal of these. They are not used anyway.

Reply:
Quote from IRC:
```
<sbueno> so maybe saying something like: "Encrypt automatically created mount points by default:"
```